qa-niaid.planx-pla.net/manifests/hatchery/hatchery.json

```diff
@@ -0,0 +1,72 @@
+{
+  "user-namespace": "jupyter-pods-qa-niaid",
+  "sub-dir": "/lw-workspace",
+  "user-volume-size": "10Gi",
+  "sidecar": {
+    "cpu-limit": "1.0",
+    "memory-limit": "256Mi",
+    "image": "quay.io/cdis/gen3fuse-sidecar:2020.08",
```

Comment:
you can use 2020.11, we're updating the versions https://github.com/uc-cdis/gitops-qa/pull/1023